rfcs/README.md

```diff
@@ -2,14 +2,58 @@
+    after being locked in a room for an hour with X other engineers.
+- A more inclusive review proces
+  - Supporting both quick and deep thinkers
+  - Allowing proposals and comments from contributors
+- Capturing decisions, allowing:
+  - Us to review recent decisions and build ontop of them
+  - Others to join the project more easily
+
+The React RFC Process sums it up nicely[1]
+
+> The RFC process is a great opportunity to get more eyeballs on your proposal before it becomes a part of a released version. Quite often, even proposals that seem "obvious" can be significantly improved once a wider group of interested people have a chance to weigh in.
+>
+> The RFC process can also be helpful to encourage discussions about a proposed feature as it is being designed, and incorporate important constraints into the design while it's easier to change, before the design has been fully implemented.
+
 ## RFC process
 
-### Creation
+### 1. Creation
+
+- Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+- Write the RFC
+- Submit a PR and label it with **Type: RFC**.
+
+### 2. Review
 
-Start by making a copy of [TEMPLATE.md](./TEMPLATE.md) under an appropriate area sub-folder and name. (While we're still experimenting with the RFC process, feel free to create a new area sub-folder if you don't think an appropriate one exists yet.)
+A key purpose of the review phase is to seek feedback to help improve your idea. You the author are ultimately accountable for the proposal and it's impact. Reviewers can, through comments, help you make the best decision you can. Throughout this review and process it is important that we strive to avoid both [Design by Committee](https://en.wikipedia.org/wiki/Design_by_committee) and [Diffusion of Responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility).
 
-Once you're done writing, make a PR and label it with **Type: RFC**.
+- Seek support and integrate feedback.
+  - It is the responsibility of the author(s) to read, consider, and respond to all feedback in a timely manner. This does not mean all feedback must be integrated into the proposal, but it must be considered.
+  - Tip: If you want to good engagement on your RFC, reserve time to respond to comments quickly.
 
-### Implementation
+Once the RFC has been open for **at least 2 days**, and recieved sufficent comments, the author can begin seeking approval.
+
+### 3. Approval
+
+We're still working through details for approval, below are the current guidelines.
+
+Following the theme of author accountability, in general it's up to the author to decide when they feel they have recieved enough feedback, and enough approvals to proceed.
+
+- When in doubt, seek **2 approvals** before merging the RFC.
+
+### 4. Implementation
```

Comment:
Might be worth mentioning that (while this practice isn't recommended) if implementation has already started on any parts of the RFC, that should be noted in the RFC text.

Reply:
Yep, it may, but let's keep an eye on whether this becomes a recurring issue. Same as what I said on the comment thread before, if this is a pattern we can update guidance, but let's not overreact and add on to the guidance based on just one instance (also adding guidance here may end up inviting implementation to start before the RFC is complete which is kind of a slippery slope)